pkgs/by-name/be/beatsabermodmanager/package.nix

```diff
@@ -8,37 +8,32 @@
   xdg-utils,
 }:
 
-buildDotnetModule rec {
+buildDotnetModule {
   pname = "beatsabermodmanager";
-  version = "0.0.5";
+  version = "0.0.7";
 
   src = fetchFromGitHub {
     owner = "affederaffe";
     repo = "BeatSaberModManager";
-    rev = "v${version}";
-    hash = "sha256-HHWC+MAwJ+AMCuBzSuR7FbW3k+wLri0B9J1DftyfNEU=";
+    # v0.0.7 was published without a release tag
+    rev = "8bf3611a8b33c95e7a0340504894cf7b46822107";
+    hash = "sha256-mRC/dGkpmKBQ2euyCOOvOkN+LUOHW1p1L/VQ4bWSUpY";
     fetchSubmodules = true; # It vendors BSIPA-Linux
   };
 
   dotnet-sdk = with dotnetCorePackages; combinePackages [
-    sdk_7_0
+    sdk_8_0
     sdk_6_0
```

Comment:
Do we still need this?

Reply:
it should, i didnt try removing it now but the submodule hasnt been updated for 2 years.
from what i understand thats the reason why this is needed